package.xml

```diff
@@ -16,15 +16,9 @@
   <depend>libcairo2-dev</depend>
   <depend>libqwt6</depend>
 
-  <build_depend>libqt4-dev</build_depend>
```

Comment:
I'd prefer these dependencies are commented and a comment is added mentioning why this was done, so if we ever want to use the GUI part, we know that we need to install the qt5 equivalents to these.

I don't have a strong opinion on this though, since I doubt we'd ever use the GUI part, if we ever did.

FYI @servos